http/basic/README.md

```diff
@@ -0,0 +1,209 @@
+# Generic HTTP Protocol Binding
+
+[![Javadocs](http://www.javadoc.io/badge/io.cloudevents/cloudevents-http-basic.svg?color=green)](http://www.javadoc.io/doc/io.cloudevents/cloudevents-http-basic)
+
+For Maven based projects, use the following to configure the CloudEvents Generic HTTP Transport:
+
+```xml
+<dependency>
+    <groupId>io.cloudevents</groupId>
+    <artifactId>cloudevents-http-basic</artifactId>
+    <version>2.0.0-milestone1</version>
+</dependency>
+```
+
+## Sending and Receiving CloudEvents
+To send and receive CloudEvents we use `MessageWriter` and `MessageReader`, respectively.
+This module offers generic factory methods for creation of those in `HttpMessageFactory`.
+
+```java
+public class HttpMessageFactory {
+    public static MessageReader createReader(Consumer<BiConsumer<String,String>> forEachHeader, byte[] body);
+    public static MessageReader createReader(Map<String,String> headers, byte[] body);
+    public static MessageReader createReaderFromMultimap(Map<String,List<String>> headers, byte[] body);
+    public static MessageWriter createWriter(BiConsumer<String, String> putHeader, Consumer<byte[]> sendBody);
+}
+```
+
+Below you can see examples of usage.
+
+### HttpServlet's Request and Response
```

Comment:
Remove all this code below for this and other http impls and point to the sample